proxy/src/bind.rs

```diff
@@ -151,30 +157,40 @@ where
             self.sensors.connect(c, &client_ctx)
         };
 
-        // Establishes an HTTP/2.0 connection
-        let client = tower_h2::client::Client::new(
+        let client = transparency::Client::new(
+            protocol,
             connect,
-            self.h2_builder.clone(),
-            ::logging::context_executor(("client", *addr), self.executor.clone()),
+            self.executor.clone(),
+            //::logging::context_executor(("client", *addr), self.executor.clone()),
         );
 
-        let h2_proxy = self.sensors.http(self.req_ids.clone(), client, &client_ctx);
+        let proxy = self.sensors.http(self.req_ids.clone(), client, &client_ctx);
 
         // Automatically perform reconnects if the connection fails.
         //
         // TODO: Add some sort of backoff logic.
-        Reconnect::new(h2_proxy)
+        Reconnect::new(proxy)
     }
 }
 
-// ===== impl Bind =====
+// ===== impl BindProtocol =====
+
 
-impl<B> control::discovery::Bind for Bind<Arc<ctx::Proxy>, B>
+impl<C, B> Bind<C, B> {
```

Comment:
Very minor, but: seems to me this should still have the `// ===== impl Bind =====` comment over it?